probe-rs/src/probe/wlink/mod.rs

```diff
@@ -187,11 +185,15 @@ impl WchLink {
 
         self.get_probe_info()?;
 
+        // this is the official version format. So "v31" is actually a 2.11
+        let version_code = self.v_major * 10 + self.v_minor;
+
         tracing::info!(
-            "WCH-Link variant: {}, firmware version: {}.{}",
+            "WCH-Link variant: {}, firmware version: {}.{} (v{})",
```

Comment:
The `v30` or `v31` code is the version format of official tools.
I can also use the official version format only. But I think it's better to provide both because the official version scheme is misleading.